src/pages/home/report/EmojiPickerMenu/dynamicEmojiSize.js

```diff
@@ -1,15 +1,8 @@
 import styleVariables from '../../../../styles/variables';
 
 const dynamicEmojiSize = (windowWidth) => {
-    if (windowWidth <= 320) {
-        return styleVariables.fontSizeSmall;
-    }
-    if (
-        windowWidth <= 480
-    ) {
-        return styleVariables.fontSizeNormal;
-    }
-
+    if (windowWidth <= 320) { return styleVariables.fontSizeSmall; }
+    if (windowWidth <= 480) { return styleVariables.fontSizeNormal; }
```

Comment:
oh sorry for the confusion, i didn't mean the whole line, just the if condition, i.e.,
```
if (windowWidth <= 480) {
    return styleVariables.fontSizeNormal;
}
```
That would match our examples in the style guide. Could you make the same changes to the if conditions in your file.

Reply:
Done